src/Extensions/Shopware/Install/Services/Install/Release.php

```diff
@@ -110,7 +110,7 @@ public function installShopware(InstallationRequest $request)
     {
         $this->releaseDownloader->downloadRelease($request->getRelease(), $request->getInstallDir());
 
-        if ($request->getRelease() === 'latest' || version_compare($request->getRelease(), '5.1.2', '>=')) {
+        if ($request->getRelease() === 'latest' || version_compare($request->getRelease(), '5.2.12', '>=')) {
```

Comment:
yeah, you are right. has something to do with the way of installation. thought it was just the latest release.